src/io/flutter/view/FlutterView.java

```diff
@@ -317,7 +316,7 @@ protected void perform(AnActionEvent event) {
   }
 }
 
-abstract class AbstractFlutterAction extends DumbAwareAction {
+abstract class AbstractFlutterAction extends AnAction {
```

Comment:
Should we does this more universally?  I admit I don't really get what `DumbAwareAction`s are about and the docs are errr, lacking. 🙁